templates/zerver/desktop_login.html

```diff
@@ -0,0 +1,18 @@
+{% extends "zerver/portico.html" %}
+{% set entrypoint = "desktop-login" %}
+
+{% block portico_content %}
+<div class="app portico-page">
+    <div class="app-main portico-page-container center-block flex full-page new-style">
+        <div class="inline-block">
+            {% trans %}
+            <h1>Finish desktop login</h1>
```

Comment:
Do we want the '<h1>' tags inside the `{% trans %}` block?

Reply:
Better to put them outside, yeah.